programs/client/Client.cpp

```diff
@@ -998,7 +999,7 @@ void Client::addAndCheckOptions(OptionsDescription & options_description, po::va
     /// Main commandline options related to client functionality and all parameters from Settings.
     options_description.main_description->add_options()
         ("config,c", po::value<std::string>(), "config-file path (another shorthand)")
-        ("host,h", po::value<std::string>()->default_value("localhost"), "server host")
+        ("host,h", po::value<std::vector<std::string>>()->multitoken()->default_value({"localhost"}, "localhost"), "list of server hosts")
```

Comment:
Please add more detailed description. For example, like in `interleave-queries-file` option

Reply:
Ok, I will make it when add parsing hosts with port (to call `clickhouse client --host localhost:9000`)

Reply:
You can add it now and when you will change for port, you can also modify description